common/nym_offline_compact_ecash/src/common_types.rs

```diff
@@ -7,15 +7,26 @@ use crate::helpers::{g1_tuple_to_bytes, recover_g1_tuple};
 use bls12_381::{G1Projective, Scalar};
 use serde::{Deserialize, Serialize};
 use subtle::Choice;
+use utoipa::ToSchema;
 
 pub type SignerIndex = u64;
 
-#[derive(Debug, Clone, Copy, PartialEq, Serialize, Deserialize)]
+#[derive(Debug, Clone, Copy, PartialEq, Serialize, Deserialize, ToSchema)]
 pub struct Signature {
+    #[schema(value_type = G1ProjectiveSchema)]
     pub(crate) h: G1Projective,
+    #[schema(value_type = G1ProjectiveSchema)]
     pub(crate) s: G1Projective,
 }
 
+#[derive(ToSchema)]
+#[schema(title = "G1Projective")]
+pub struct G1ProjectiveSchema {
```

Comment:
I dont think we want that schema for the purposes of any api. if we're returning raw signature anywhere, they're using json serialiser and thus stringified, I think hex, encoding for G1 points

Reply:
moved `ToSchema`  derives out of ecash crate, as discussed